### PR TITLE
minor update to log message in browser command

### PR DESCRIPTION
**Pull request by pgfox**
changed log message to state browse/browsing instead of consume/consuming

### CODE REVIEW

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ public void browse() {
          }
       }
 
-      System.out.println(threadName + " Consumer thread finished");
+      System.out.println(threadName + " thread finished");
```
Comment (michaelandrepearce) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java]: Browser thread finished

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java [hunk 2]
```diff
@@ -115,7 +115,7 @@ public void browse() {
             finished.countDown();
          }
          if (consumer != null) {
-            System.out.println(threadName + " Consumed: " + this.getReceived() + " messages");
+            System.out.println(threadName + " browsed: " + this.getReceived() + " messages");
```
Comment (michaelandrepearce) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java]: capitalise "Browsed: " to match other capitalisation in other sys outs.

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java [hunk 3]
```diff
@@ -81,7 +81,7 @@ public void browse() {
          while (enumBrowse.hasMoreElements()) {
             Message msg = enumBrowse.nextElement();
             if (msg != null) {
-               System.out.println(threadName + " Received " + (msg instanceof TextMessage ? ((TextMessage) msg).getText() : msg.getJMSMessageID()));
+               System.out.println(threadName + " Browsing " + (msg instanceof TextMessage ? ((TextMessage) msg).getText() : msg.getJMSMessageID()));
```
Comment (michaelandrepearce) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java]: consistency of casing, maybe lowercase Browsing? either that or make others Capitalised.

**Comment by michaelandrepearce** [comment]
@pgfox can you open a JIRA for this and update the commit message to include it, e.g.

ARTEMIS-XXXX changed log message to state browse/browsing instead of consume/consuming
 
where xxxx is the jira number.

**Comment by clebertsuconic** [comment]
@michaelandrepearce I was going to merge this with NO-JIRA ammended

**Comment by michaelandrepearce** [comment]
@clebertsuconic cool beans. :)

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java (hunk 3 above)
Comment (clebertsuconic) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java]: @pgfox you think you can change that?

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java (hunk 1 above)
Comment (clebertsuconic) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java]: +1

**Comment by pgfox** [comment]
@clebertsuconic  @michaelandrepearce  Sure, your suggested changes should be in the pull request now. Thanks

**Pull request closed without merging**